server/planning/content_api/types/planning.py

```diff
@@ -0,0 +1,59 @@
+from superdesk.core.resources import fields, Dataclass
+from pydantic import Field
+from superdesk.utc import utcnow
+from datetime import datetime
+from planning.types import PlanningResourceModel, PLANNING_EVENT_LINK_METHOD
+
+
+class AgendaItem(Dataclass):
+    qcode: fields.Keyword
+    name: str
+
+
+class MatchingProduct(Dataclass):
+    code: fields.Keyword
+    name: str
+
+
+class RelatedEvent(Dataclass):
+    _id: fields.Keyword
```

Comment:
The fields here are `rel`, `uri`, `literal` and `name` [see code](https://github.com/superdesk/superdesk-planning/blob/async/server/planning/output_formatters/json_planning.py#L118).